tests/test_tank.py

```diff
@@ -150,8 +150,8 @@ def test_mass_based_tank():
     real_geometry = TankGeometry(
         {
             (0, 0.0559): bottom_endcap,
-            (0.0559, 0.7139): lambda y: 0.0744,
-            (0.7139, 0.7698): top_endcap,
+            (0.0559, 0.8039): lambda y: 0.0744,
+            (0.8039, 0.8698): top_endcap,
```

Comment:
Comment on these changes: after making the assertion from lines 192-195, the tests failed due to the fact that the defined geometry volume is too low. Therefore the tank ended up being overfilled and had to be increased in size.